vars/Debian.yml

```diff
@@ -1,3 +1,4 @@
 ---
 __unattended_origins_patterns:
-  - 'origin=Debian,archive=${distro_codename},label=Debian-Security'
+  - 'origin=Debian,archive=stable,label=Debian-Security'
+  - 'origin=Debian,archive=oldstable,label=Debian-Security'
```

Comment:
But I don't know If we'll do dist-upgrade instead of rebuild a new server. So in that case it doesn't make sense to have ```oldstable```.